histomicstk/cli/NucleiDetection/NucleiDetection.py

```diff
@@ -154,8 +304,12 @@ def main(args):
 
     it_kwargs = {
         'tile_size': {'width': args.analysis_tile_size},
-        'scale': {'magnification': args.analysis_mag},
+        'scale': {'magnification': args.analysis_mag}
     }
+    try:
+        it_kwargs['frame'] = int(args.frame)
```

Comment:
We should be able to get rid of this try/except block by doing 
```
if getattr(args, 'frame', None) is not None:
    it_kwargs['frame'] = int(args.frame)
```
This will allow legitimate errors to still throw (such as someone oddly passing a non-number to frame).

Reply:
I tried this method, but the `getattr()` cannot have None assigned to it's output. So, i've changed the code a bit to check if `args.frame` exist and is a digit. Otherwise it throws an exception.
`if not args.frame or args.frame.startswith('{#control'):
        args.frame = None
    elif not args.frame.isdigit():
        raise Exception("The given frame value is not an integer")
    else:
        it_kwargs['frame'] = args.frame`